fixieai/client/client.py

```diff
@@ -211,6 +212,140 @@ def refresh_agent(self, agent_handle: str):
             headers=self._request_headers,
         ).raise_for_status()
 
+    def create_agent_revision(
+        self,
+        handle: str,
+        make_current: bool,
+        *,
+        reindex_corpora: bool = False,
+        metadata: Optional[Dict[str, str]] = None,
+        external_url: Optional[str] = None,
+        python_gzip_tarfile: Optional[BinaryIO] = None,
+    ) -> str:
+        """Creates a new Agent revision."""
```

Comment:
Would you mind adding "Args:" to the docstring explaining all of the exciting options?

Reply:
Good call, added.